docs/source/api.rst

```diff
@@ -244,3 +248,12 @@ Read Buffering
 
 .. autoclass:: fsspec.caching.BlockCache
    :members:
+
+Utilities
+---------
+
+.. autosummary::
+
+   fsspec.utils.read_block
+
+.. autofunction:: fsspec.utils.read_block
```

Comment:
This function is referred to elsewhere, so it is useful to include its docstring here. I haven't included docstrings of any other `utils` functions as many are effectively private.